documentation/asciidoc/accessories/camera/filters.adoc

```diff
@@ -30,47 +30,41 @@ The HQ and GS Cameras use a Hoya CM500 infrared filter. Its transmission charact
 
 image::images/hoyacm500.png[CM500 Transmission Graph,width="65%"]
 
-== Filter Removal
+== IR Filter
 
-NOTE: This procedure applies to both the HQ and GS cameras. 
-
-WARNING: *This procedure cannot be reversed:* the adhesive that attaches the filter will not survive being lifted and replaced, and while the IR filter is about 1.1mm thick, it may crack when it is removed. *Removing it will void the warranty on the product*. Nevertheless, removing the filter will be desirable to some users.
+Both the High Quality Camera and Global Shutter Camera contain an IR filter, which is used to reduce the camera's sensitivity to infrared light. This ensures that outdoor photos look more natural. However, some nature photography can be enhanced with the removal of this filter; the colours of sky, plants, and water can be affected by its removal. The camera can also be used without the filter for night vision in a location that is illuminated with infrared light.
 
-image:images/FILTER_ON_small.jpg[width="65%"]
+=== Filter Removal
 
-Both the High Quality Camera and Global Shutter Camera contain an IR filter, which is used to reduce the camera's sensitivity to infrared light. This ensures that outdoor photos look more natural. However, some nature photography can be enhanced with the removal of this filter; the colours of sky, plants, and water can be affected by its removal. The camera can also be used without the filter for night vision in a location that is illuminated with infrared light.
+The filter removal procedure applies to both the HQ and GS cameras. The HQ camera is shown in the demonstration below.
```

Comment:
Suggestion: the 'filter removal procedure' sounds like something you ought to ask a doctor about. maybe fewer word do trick?

```suggestion
You can remove the filter from both the HQ and GS cameras. The HQ camera is shown in the demonstration below.
```